src/styles/table.styl

```diff
@@ -119,6 +120,17 @@
 
 mime-types('fil-file-', '../assets/icons/')
 
+.fil-file-preview
+    content ''
+    position absolute
+    left 0
+    top 50%
+    transform translateY(-50%)
```

Comment:
Yeah, the pushing it down with `top` then pushing it up again with `transform` feels hacky, but alright. How does it compare to something [like this](https://codepen.io/anon/pen/BZxgBJ)? The same but less verbose?